packages/grafana-toolkit/src/plugins/types.ts

```diff
@@ -98,4 +98,7 @@ export interface ManifestInfo {
   plugin: string;
   version: string;
   files: Record<string, string>;
+  toolkit?: {
```

Comment:
Why optional? Isn't it always set with these changes?

Reply:
@marefr I made it optional so that the type can be used without it and it is also optional on the rest api side. See: https://github.com/grafana/grafana/blob/main/packages/grafana-toolkit/src/plugins/manifest.ts#L37-L41 and https://github.com/grafana/grafana-com/pull/3655/files#diff-12b3aade157e61f049d0b65be9d1ecbd85dfabc37c6d960e8697bc2f29064145R456

In general it should not really a part of the `manifest` itself, as we do not write it into the generated manifest file after calling the api. I just did not want to create a whole separate type for it for now, maybe when we go more into improving the toolkit we can optimize it.

Reply:
Make sense